webapp/exercise_admin/views.py

```diff
@@ -140,3 +142,55 @@ def get_feedback_questions(request):
     return JsonResponse({
         "result": result
     })
+
+def create_feedback_question(request):
+    if request.method != "POST":
+        return HttpResponseNotAllowed(["POST"])
+
+    if not (request.user.is_staff and request.user.is_authenticated() and request.user.is_active):
+        return JsonResponse({
+            "error" : {
+                "__all__" : {
+                    "message" : "Only logged in users can send feedback!",
+                    "code" : "authentication"
+                }
+            }
+        })
+
+    data = request.POST.dict()
+    data.pop("csrfmiddlewaretoken")
+    choice_count = len([k for k in data.keys() if k.startswith("choice_field")])
+    
+    form = CreateFeedbackQuestionForm(choice_count, data)
+    
+    if form.is_valid():
+        question = form.cleaned_data["question_field"]
+        question_type = form.cleaned_data["type_field"]
+        choices = [v for (k, v) in form.cleaned_data.items() if k.startswith("choice_field")]
+        
+        if question_type == "THUMB_FEEDBACK":
+            q_obj = ThumbFeedbackQuestion(question=question, slug="")
```

Comment:
Doesn't .save() already handle the `slug`? Shouldn't this work (better) without the `slug=""`?